src/golang/lib/job/spec.go

```diff
@@ -279,6 +294,35 @@ func NewParamSpec(
 	}
 }
 
+func NewSystemMetricSpec(
+	name string,
+	storageConfig *shared.StorageConfig,
+	metadataPath string,
+	metricName string,
+	inputContentPaths []string,
+	inputMetadataPaths []string,
+	outputContentPaths []string,
+	outputMetadataPaths []string,
+	outputArtifactTypes []artifact.Type,
+) Spec {
+	return &SystemMetricSpec{
+		basePythonSpec: basePythonSpec{
+			baseSpec: baseSpec{
+				Type: SystemMetricJobType,
+				Name: name,
+			},
+			StorageConfig: *storageConfig,
+			MetadataPath:  metadataPath,
+		},
+		InputContentPaths:   inputContentPaths,
```

Comment:
whats the point of the input content path?

Reply:
I removed it